app/controllers/sessions_controller.rb

```diff
@@ -8,7 +8,7 @@ def create
     user.google_refresh_token = refresh_token if refresh_token.present?
     user.save
     session[:user_id] = user.id
-    redirect_to '/user'
```

Comment:
Good thinking, makes more sense to have a dashboard_controller than making user/show.